### ISSUE TITLE
Modrinth command failed

### ISSUE THREAD

**Issue by yurisasc** [itzg/docker-minecraft-server#1965]
I only recently got this issue yesterday. It was working normally and I had not changed the list of mods to be downloaded from Modrinth. Here's the server log

<details>
<summary>Server log</summary>

> 2023-02-16T16:20:43.831+10:00 | [init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 10 1000 1000 6144 Feb 16 00:59 /data'
-- | --
  | 2023-02-16T16:20:46.504+10:00 | [init] Resolved version given LATEST into 1.19.3 and major version 1.19
  | 2023-02-16T16:20:47.237+10:00 | [init] Resolving type given FABRIC
  | 2023-02-16T16:20:51.142+10:00 | [init] Downloading datapack https://www.dropbox.com/s/z7xv1sr7f4fleqv/Vein%20miner.zip?dl=1 ...
  | 2023-02-16T16:20:54.790+10:00 | [init] Downloading datapack https://www.dropbox.com/s/x3bx5pvf9zjqhh5/vanillarefresh-v1-4-8c.zip?dl=1 ...
  | 2023-02-16T16:20:58.482+10:00 | [init] Downloading datapack https://www.dropbox.com/s/zdrwwd6fer57efh/pk-waystones-v-1-0-5.zip?dl=1 ...
  | 2023-02-16T16:21:01.946+10:00 | [init] Downloading datapack https://www.dropbox.com/s/w5hmtyh73juqq97/BlazeandCaves%20Advancements%20Pack%201.15.3.zip?dl=1 ...
  | 2023-02-16T16:21:06.675+10:00 | [init] Downloading datapack https://www.dropbox.com/s/u6eeyul62vfxexg/Health%20Indications%20v2.2%201.19.zip?dl=1 ...
  | 2023-02-16T16:21:10.261+10:00 | [init] Downloading datapack https://www.dropbox.com/s/t87nhnc39t2nkjr/new-in-town-by-kanokarob-v2.14.zip?dl=1 ...
  | 2023-02-16T16:21:14.344+10:00 | [init] Downloading datapack https://www.dropbox.com/s/tt0xz19z2405r1w/custom-recipe-tools.zip?dl=1 ...
  | 2023-02-16T16:21:17.826+10:00 | [init] Downloading mod/plugin https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/fabric/build/libs/Geyser-Fabric.jar ...
  | 2023-02-16T16:21:28.685+10:00 | [mc-image-helper] 16:21:28.682 INFO : Downloading fabric-api-0.73.2+1.19.3.jar
  | 2023-02-16T16:21:29.386+10:00 | [mc-image-helper] 16:21:29.385 INFO : Downloading lithium-fabric-mc1.19.3-0.10.4.jar
  | 2023-02-16T16:21:30.033+10:00 | [mc-image-helper] 16:21:30.033 INFO : Downloading lazydfu-0.1.3.jar
  | 2023-02-16T16:21:30.694+10:00 | [mc-image-helper] 16:21:30.694 INFO : Downloading ferritecore-5.1.0-fabric.jar
  | 2023-02-16T16:21:31.318+10:00 | [mc-image-helper] 16:21:31.318 INFO : Downloading starlight-1.1.1+fabric.ae22326.jar
  | 2023-02-16T16:21:32.824+10:00 | [mc-image-helper] 16:21:32.824 INFO : Downloading memoryleakfix-1.19.3-0.7.0.jar
  | 2023-02-16T16:21:34.690+10:00 | [mc-image-helper] 16:21:34.690 INFO : Downloading mods-command-mc1.19.3-1.0.12.jar
  | 2023-02-16T16:21:36.128+10:00 | [mc-image-helper] 16:21:36.128 INFO : Downloading tabtps-fabric-mc1.19.3-1.3.16.jar
  | 2023-02-16T16:21:36.751+10:00 | [mc-image-helper] 16:21:36.751 INFO : Downloading fabrictailor-2.0.2.jar
  | 2023-02-16T16:21:44.435+10:00 | [mc-image-helper] 16:21:44.433 ERROR : 'modrinth' command failed. Version is 1.25.3
  | 2023-02-16T16:21:44.435+10:00 | io.netty.handler.timeout.ReadTimeoutException: null
</details>

Here's the environment variables I enabled

<details>
<summary>Environment variables</summary>

```
{
   "EULA":"TRUE",
   "MEMORY":"4G",
   "TYPE":"FABRIC",
   "DIFFICULTY":"normal",
   "ONLINE_MODE":"FALSE",
   "MOTD":"Hello World!",
   "MAX_PLAYERS":"5",
   "VIEW_DISTANCE":"10",
   "MODE":"survival",
   "ENABLE_COMMAND_BLOCK":"TRUE",
   "TZ":"Australia/Brisbane",
   "MODRINTH_PROJECTS":"fabric-api,lithium,lazydfu,ferrite-core,starlight,memoryleakfix,mods-command,tabtps,fabrictailor,no-chat-reports,convenient-mobgriefing",
   "MODRINTH_DOWNLOAD_OPTIONAL_DEPENDENCIES":"TRUE",
   "MODS":"https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/fabric/build/libs/Geyser-Fabric.jar", 
}
```
</details>

The error was gone when I removed `no-chat-reports` from the `MODRINTH_PROJECTS`, but restarting the server (with `no_chat_reports` still removed) caused the error again.

**Comment by yurisasc** [comment]
Yup! Can confirm it helps. Thanks for the swift response. So, maybe the default timeout value was not enough for my server. To give more context, I deployed the server on AWS ECS in `ap-southeast-3` region.

**Comment by visualcookie** [comment]
This issue seems to be back. After running `docker compose restart` I get the following in my logs:

![image](https://user-images.githubusercontent.com/1857554/231592016-717945b8-af24-4aac-a67d-2afb3f61c445.png)

**Comment by VJmes** [comment]
I'm having a similar issue with Modrinth that has appeared in the last few days:
`[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 9 1000 1000 4096 Apr 13 10:04 /data'
[init] Resolved version given LATEST into 1.19.4 and major version 1.19
[init] Autopause functionality enabled
[init] Resolving type given FABRIC
[mc-image-helper] 07:20:17.208 ERROR : 'modrinth' command failed. Version is 1.27.4
me.itzg.helpers.files.ManifestException: Failed to load existing manifest from /data/.modrinth-manifest.json
`

**Comment by ddattee** [comment]
Hi,

same here I get this error since this last week.
I'm guessing that's a problem with a mod from Modrinth but I can't find which one is at fault.

Here is the error I get ( I don't always get it after the same mod downloading when I wathc the logs)  :
```
[mc-image-helper] 23:37:03.778 ERROR : 'modrinth' command failed. Version is 1.38.16
me.itzg.helpers.http.FailedRequestException: HTTP request of https://api.modrinth.com/v2/version/bg7gh7bA failed with 404 Not Found: Fetching object content
```

And here is my env : 
```
EULA="TRUE"
TYPE=FABRIC
DEBUG="false"
VERSION="1.20.4"
INIT_MEMORY=4G
MAX_MEMORY=8G
ENABLE_WHITELIST="true"
ENABLE_RCON=true
RCON_PASSWORD=***
VIEW_DISTANCE=16
SIMULATION_DISTANCE=14
DIFFICULTY=hard
MODRINTH_PROJECTS=fabric-api,lithium,sodium,simple-voice-chat:beta,audioplayer:alpha,view-distance-fix:alpha,enhanced-groups:alpha,xaeros-world-map,xaeros-minimap,dynmap:beta,cloth-config,luckperms,viafabricplus,worldedit,starlight,discord-mc-chat:2.3.1,spark,krypton,ferrite-core,memoryleakfix,moreculling,carpet,carpet-fixes,forge-config-api-port:v20.4.3-1.20.4-Fabric,yawp:bg7gh7bA,comforts
```
